proto/ssn/async/v1/async.proto

```diff
@@ -39,6 +39,9 @@ service TransactionService {
   rpc DeleteTransaction(DeleteTransactionRequest) returns (google.protobuf.Empty) {
     option (google.api.http) = {delete: "/v1/transactions/{id}"};
   }
+  rpc DeleteTags(DeleteTagsRequest) returns (google.protobuf.Empty) {
```

Comment:
I think it should be like https://github.com/e-conomic/vmlapis/blob/master/proto/asgt/v2/dataset_service.proto#L46